mteb/leaderboard/app.py

```diff
@@ -1020,30 +1019,3 @@ def update_tables(
     total_time = time.time() - app_start
     logger.info(f"=== Leaderboard app initialization complete in {total_time:.2f}s ===")
     return demo
-
-
-if __name__ == "__main__":
-    logging.getLogger("mteb.load_results.task_results").setLevel(
-        logging.ERROR
-    )  # Warnings related to task split
-    logging.getLogger("mteb.model_meta").setLevel(
-        logging.ERROR
-    )  # Warning related to model metadata (fetch_from_hf=False)
-    logging.getLogger("mteb.load_results.benchmark_results").setLevel(
-        logging.ERROR
-    )  # Warning related to model metadata (fetch_from_hf=False)
-    warnings.filterwarnings("ignore", message="Couldn't get scores for .* due to .*")
-
-    app = get_leaderboard_app()
-
-    head = """
-    <link href="https://cdn.jsdelivr.net/npm/tailwindcss@2.2.19/dist/tailwind.min.css" rel="stylesheet">
-    """
-    app.launch(
-        server_name="0.0.0.0",
-        server_port=7860,
-        theme=gr.themes.Soft(
-            font=[gr.themes.GoogleFont("Roboto Mono"), "Arial", "sans-serif"],
-        ),
-        head=head,
-    )
```

Comment:
Isn't removing this a breaking change - I don't think we can do that (we could add a deprecation warning to suggest the new workflow)

Reply:
Yeah we should leave this untouched.

Reply:
reverted